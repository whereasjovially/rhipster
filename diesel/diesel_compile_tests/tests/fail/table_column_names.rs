#[macro_use] extern crate diesel;

table! {
    users {
        id -> Integer,
        users -> Integer,
    }
}
// error-pattern: Column `users` cannot be named the same as its table.
// error-pattern: You may use `#[sql_name = "users"]` to reference the table's `users` column.

fn main() {}
