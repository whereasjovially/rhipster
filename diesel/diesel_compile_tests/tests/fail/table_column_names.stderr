error: Column `users` cannot be named the same as its table.
 You may use `#[sql_name = "users"]` to reference the table's `users` column.
 Docs available at: `https://docs.diesel.rs/diesel/macro.table.html`

 --> $DIR/table_column_names.rs:3:1
  |
3 | / table! {
4 | |     users {
5 | |         id -> Integer,
6 | |         users -> Integer,
7 | |     }
8 | | }
  | |_^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
